### PR TITLE
[[ Bug 19981 ]] Fix inability to use custom fonts on Window

### PR DESCRIPTION
**Pull request by livecodeian**

### CODE REVIEW

File: engine/src/w32dc.cpp [hunk 1]
```diff
@@ -157,6 +157,9 @@ bool MCScreenDC::loadfont(MCStringRef p_path, bool p_globally, void*& r_loaded_f
 	if (t_success && p_globally)
 		PostMessage(HWND_BROADCAST, WM_FONTCHANGE, 0, 0);
     
+	if (t_success)
```
Comment (runrevmark) [inline engine/src/w32dc.cpp]: Do we need to add it to a custom font collection if 'globally'?

**Comment by runrevmark** [comment]
@livecodeian: This looks great - the only question is about the need for a custom collection if 'globally' is used (as mentioned in the comment).

**Comment by runrevmark** [comment]
@livecode-vulcan review ok 6655641

**Comment by livecode-vulcan** [comment]
💙 review by @runrevmark ok 66556415e2913a07270576351b2586470aa5ea3d

**Comment by livecode-vulcan** [comment]
😎 test success 66556415e2913a07270576351b2586470aa5ea3d

* try-community-armv6-android-api8: [success](https://vulcan.livecode.com/builders/try-community-armv6-android-api8/builds/564)
* try-community-armv6-android-api9: [success](https://vulcan.livecode.com/builders/try-community-armv6-android-api9/builds/579)
* try-community-js-emscripten-sdk1.35: [success](https://vulcan.livecode.com/builders/try-community-js-emscripten-sdk1.35/builds/553)
* try-community-universal-ios-iphoneos10.3: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphoneos10.3/builds/496)
* try-community-universal-ios-iphonesimulator10.3: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphonesimulator10.3/builds/499)
* try-community-universal-mac-macosx10.6: [success](https://vulcan.livecode.com/builders/try-community-universal-mac-macosx10.6/builds/559)
* try-community-universal-mac-macosx10.9: [success](https://vulcan.livecode.com/builders/try-community-universal-mac-macosx10.9/builds/577)
* try-community-x86-linux-debian7: [success](https://vulcan.livecode.com/builders/try-community-x86-linux-debian7/builds/563)
* try-community-x86-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86-linux-debian8/builds/579)
* try-community-x86_64-linux-debian7: [success](https://vulcan.livecode.com/builders/try-community-x86_64-linux-debian7/builds/562)
* try-community-x86_64-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86_64-linux-debian8/builds/572)
* try-community-x86-win32: [success](https://vulcan.livecode.com/builders/try-community-x86-win32/builds/594)
* try-community-x86_64-win32: [success](https://vulcan.livecode.com/builders/try-community-x86_64-win32/builds/558)

**Pull request merged**